READY_TO_COMMIT.md

```
@@ -0,0 +1,251 @@
+| 速率限制 | 1000/min | 1000/min | ✅ |
+| 请求去重 | 1000 | 1000 | ✅ |
+| k6 压测 | 1000 VU | 1000 VU | ✅ |
+| 资源优化 | 轻量级 | 轻量级 | ✅ |
+
+---
+
+## 📊 待提交的更改
+
+### Git 状态
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)